### PR TITLE
Added ability to identify flow with Argo template id

### PR DESCRIPTION
**Pull request by cavandervoort**

### CODE REVIEW

File: metaflow/plugins/argo/argo_live_run.py [hunk 1]
```diff
@@ -34,13 +34,25 @@ def __init__(
         alt_flow_id_info: dict
             Alternative identifying information of flow to trigger run
         """
+        # confirm that a flow_name or template_name exists w/o conflict
         if alt_flow_id_info is None:
             alt_flow_id_info = {}
 
+        template_name = None
+        if 'template_name' in alt_flow_id_info:
+            template_name = alt_flow_id_info['template_name']
```
Comment (cloudw) [inline metaflow/plugins/argo/argo_live_run.py:43]: minor: I like the explicitly of your code. Looks like overall you are trying to do `self._template_name = alt_flow_id_info.get("template_name", flow_name.lower())` with some additional checks.

File: metaflow/plugins/argo/argo_live_run.py [hunk 2]
```diff
@@ -129,6 +142,7 @@ def trigger(
                         f"Time waited: {int((time.time() - start_time)/60)}"
                         f" minutes out of a possible {wait_timeout}"
                     )
+                    self._print_status()
```
Comment (cloudw) [inline metaflow/plugins/argo/argo_live_run.py]: Not necessarily in this PR, but the `trigger` function looks quite verbose to me now with status printed every 5 seconds. An ML pipeline can be ran for hours, and having too much printed can make error log hard to read.

Can you add a `verbose` parameter to the function, and print less info when `verbose` is `False`?

**Review by cloudw — APPROVED** [review]

File: metaflow/plugins/argo/argo_live_run.py (hunk 1 above)
Comment (cavandervoort) [reply metaflow/plugins/argo/argo_live_run.py:43]: ok leave as is

File: metaflow/plugins/argo/argo_live_run.py (hunk 2 above)
Comment (cavandervoort) [reply metaflow/plugins/argo/argo_live_run.py]: add verbose for this feature (see also, Yun's FTF kfp feature)

Comment (cavandervoort) [reply metaflow/plugins/argo/argo_live_run.py]: change: remove _print_status from use in feature, but keep for my debugging

Comment (cavandervoort) [reply metaflow/plugins/argo/argo_live_run.py]: done

**Pull request merged**